components/Header/Menu/RoomMenu.vue

```diff
@@ -19,10 +19,10 @@
                 {{ refreshingInvite ? 'Refreshing...' : refreshInviteTooltip }}
             </MenuOption>
         </MenuSection>
-        <MenuOption v-if="room.portal.status === 'open'" name="restartPortal" icon="cube" :loading=restartingPortal :disabled=restartingPortal>
+        <MenuOption v-if="room.portal.status === 'open' && isRoomOwner" name="restartPortal" icon="cube" :loading=restartingPortal :disabled=restartingPortal>
```

Comment:
Nitpick: put a newline before each attribute on long elements like these. 

```suggestion
<MenuOption 
    v-if="room.portal.status === 'open' && isRoomOwner"
    name="restartPortal" 
    icon="cube" 
    :loading=restartingPortal 
    :disabled=restartingPortal>
```

Reply:
That's what I've been doing since before as I feel it is more readable like this, but @neoncloth didn't agree with it and pretty much reverted it.

Reply:
ahhh, well he's our PM so we'll go with it.